.golangci.yml

```diff
@@ -0,0 +1,71 @@
+run:
+  # Timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 3m
+
+  # Exit code when at least one issue was found, default is 1
+  issues-exit-code: 1
+
+  # Include test files or not, default is true
+  tests: true
+
+linters:
+  # Enable specific linters
+  enable:
```

Comment:
as discussed, just to note it down, going forward we might want to relax these lists and enable further checks. For now I would say it's fine. Thanks!

Reply:
Right. But some of the linters suggested things that didn't make sense. I tested each and every one and kept the ones whose errors could be fixed or whose error was important.